### ISSUE TITLE
Get touches working.

### ISSUE THREAD

**Issue by Luka967** [Luka967/Cigar#6]
This will be done last.

**Comment by FirstCodeReviewer** [comment]
RowanHarley implemented mobile touches.
https://github.com/RowanHarley/agario

**Comment by Niall7459** [comment]
when will be complete?

**Comment by Niall7459** [comment]
just copy cigar
will be fine

**Comment by SomeoneCares11** [comment]
Yes, please - fine for me too, if anyone can help luka by putting pull request! 🙏🏻

**Comment by Niall7459** [comment]
@SomeoneCares11 Il give it ago in a min

**Comment by Niall7459** [comment]
ahh

**Comment by SomeoneCares11** [comment]
@Niall7459 i saw ur game, u did nice job to make it working, congrats

**Comment by Niall7459** [comment]
yeah its good But i want to upgrade to This Cigar. But of course touches are all its missing...

**Comment by SomeoneCares11** [comment]
@Niall7459 oh, you're using the old cigar. my mistake.

**Comment by Niall7459** [comment]
yeah sorry.

**Comment by SuperOP535** [comment]
This has been implemented at Cigar2 https://github.com/Cigar2/Cigar2
